controllers/DefaultController.php

```diff
@@ -94,6 +95,26 @@ public function actionView($tag = null, $panel = null)
         ]);
     }
 
+	public function actionReRun($tag)
+	{
+		$this->loadData($tag);
+
+		$requstPanel = $this->module->panels['request'];
+
+		$headers = $requstPanel->data['requestHeaders'];
+		unset($headers['content-length']);
+
+		$client = new Client();
```

Comment:
Regular curl makes more sense in this case because debug toolbar itself should not raise requirements of the application.